src/index.html

```diff
@@ -22,6 +22,394 @@
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+          </div>
+          <div class="card__rate-count">Reviews: 5</div>
+        </div>
+        <div class="card__price">
+          <div class="card__price-title">Price:</div>
+          <div class="card__price-value">$2,199</div>
+        </div>
+        <div class="card__buy-button">
+          <a
+            href="#"
+            class="card__buy-button-link"
+          >
+            Buy
+          </a>
+        </div>
+      </div>
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          class="card__image"
+          alt="product photo"
+        />
+        <div class="card__info">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__code">Product code: 195434</div>
+        <div class="card__rate">
+          <div class="card__rate-stars">
+            <div class="stars stars--4 stars--card">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+          </div>
+          <div class="card__rate-count">Reviews: 5</div>
+        </div>
+        <div class="card__price">
+          <div class="card__price-title">Price:</div>
+          <div class="card__price-value">$2,199</div>
+        </div>
+        <div class="card__buy-button">
+          <a
+            href="#"
+            class="card__buy-button-link"
+          >
+            Buy
+          </a>
+        </div>
+      </div>
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          class="card__image"
+          alt="product photo"
+        />
+        <div class="card__info">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__code">Product code: 195434</div>
+        <div class="card__rate">
+          <div class="card__rate-stars">
+            <div class="stars stars--4 stars--card">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+          </div>
+          <div class="card__rate-count">Reviews: 5</div>
+        </div>
+        <div class="card__price">
+          <div class="card__price-title">Price:</div>
+          <div class="card__price-value">$2,199</div>
+        </div>
+        <div class="card__buy-button">
+          <a
+            href="#"
+            class="card__buy-button-link"
+          >
+            Buy
+          </a>
```

Comment:
Using an anchor tag inside a button is not semantically correct. Consider using either a button element or an anchor tag, but not both. This is important for accessibility and proper HTML structure.